projects/kit/components/slider/slider.style.less

```diff
@@ -29,18 +28,13 @@
     The most left point of the track is the most left point of the thumb (at the zero-position).
     The first tick should start at the center of thumb to set correct position for all other ticks.
     */
-    @first-tick-offset: ((@thumb-width - @ticks-thickness) / 2);
+    @first-tick-offset: calc((@thumb-width - var(--t-tick-thickness)) / 2);
     @ticks-background-size: calc(100% - @thumb-width);
 
     @hide-first-tick: linear-gradient(
         to right,
-        @thumb-color 0px (@first-tick-offset + @ticks-thickness),
-        transparent @ticks-thickness
-    );
-    @ticks-gradient: repeating-linear-gradient(
```

Comment:
I wonder if `@hide-first-tick` is required if `repeating-linear-gradient` was replaced by `linear-gradient` ? 🤔